### PR TITLE
blog: Release Candidate article

### PR DESCRIPTION
**Pull request by Atinux**
- [ ] Update link of release note

### CODE REVIEW

File: content/en/announcements/6.nuxt3-rc.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+Nitro takes Nuxt to another level with multiple features:
+
+- Hot module replacement for the [server/ directory](https://v3.nuxtjs.org/guide/features/server-routes)
+- Server routes with file system similar as the pages/ directory
+- Portable and compact deployments without node\_modules dependency
+- Route caching and static pre-rendering with built-in crawler
+- Code-splitting and async chunk loading for fast server startup time
+- [And much more…](https://nitro.unjs.io/)
+
+**This means that Nuxt 3 and everything you use in your project are now [devDependencies**](https://nodejs.dev/learn/npm-dependencies-and-devdependencies)**, Nitro will take care of code-splitting and bundling everything into a portable and compact `.output` directory ready to be deployed on [any hosting providers](https://nitro.unjs.io/deploy/).
```
Comment (clemcode) [inline content/en/announcements/6.nuxt3-rc.md:101]:
```suggestion
**This means that Nuxt 3 and everything you use in your project are now [devDependencies](https://nodejs.dev/learn/npm-dependencies-and-devdependencies)**, Nitro will take care of code-splitting and bundling everything into a portable and compact `.output` directory ready to be deployed on [any hosting providers](https://nitro.unjs.io/deploy/).
```

File: content/es/announcements/6.nuxt3-rc.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+Nitro takes Nuxt to another level with multiple features:
+
+- Hot module replacement for the [server/ directory](https://v3.nuxtjs.org/guide/features/server-routes)
+- Server routes with file system similar as the pages/ directory
+- Portable and compact deployments without node\_modules dependency
+- Route caching and static pre-rendering with built-in crawler
+- Code-splitting and async chunk loading for fast server startup time
+- [And much more…](https://nitro.unjs.io/)
+
+**This means that Nuxt 3 and everything you use in your project are now [devDependencies**](https://nodejs.dev/learn/npm-dependencies-and-devdependencies)**, Nitro will take care of code-splitting and bundling everything into a portable and compact `.output` directory ready to be deployed on [any hosting providers](https://nitro.unjs.io/deploy/).
```
Comment (clemcode) [inline content/es/announcements/6.nuxt3-rc.md:101]:
```suggestion
**This means that Nuxt 3 and everything you use in your project are now [devDependencies](https://nodejs.dev/learn/npm-dependencies-and-devdependencies)**, Nitro will take care of code-splitting and bundling everything into a portable and compact `.output` directory ready to be deployed on [any hosting providers](https://nitro.unjs.io/deploy/).
```

File: content/fr/announcements/6.nuxt3-rc.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+Nitro takes Nuxt to another level with multiple features:
+
+- Hot module replacement for the [server/ directory](https://v3.nuxtjs.org/guide/features/server-routes)
+- Server routes with file system similar as the pages/ directory
+- Portable and compact deployments without node\_modules dependency
+- Route caching and static pre-rendering with built-in crawler
+- Code-splitting and async chunk loading for fast server startup time
+- [And much more…](https://nitro.unjs.io/)
+
+**This means that Nuxt 3 and everything you use in your project are now [devDependencies**](https://nodejs.dev/learn/npm-dependencies-and-devdependencies)**, Nitro will take care of code-splitting and bundling everything into a portable and compact `.output` directory ready to be deployed on [any hosting providers](https://nitro.unjs.io/deploy/).
```
Comment (clemcode) [inline content/fr/announcements/6.nuxt3-rc.md:101]:
```suggestion
**This means that Nuxt 3 and everything you use in your project are now [devDependencies](https://nodejs.dev/learn/npm-dependencies-and-devdependencies)**, Nitro will take care of code-splitting and bundling everything into a portable and compact `.output` directory ready to be deployed on [any hosting providers](https://nitro.unjs.io/deploy/).
```

**Comment by Atinux** [comment]
Sure @kazupon 

But we will release it in an hour from now

File: content/en/announcements/6.nuxt3-rc.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
+---
+template: post
+title: Annoucing Nuxt 3 Release Candidate
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:3]:
```suggestion
title: Announcing Nuxt 3 Release Candidate
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+    avatarUrl: https://pbs.twimg.com/profile_images/1370286658432724996/ZMSDzzIi_400x400.jpg
+    link: https://twitter.com/clemcodes
+tags:
+  - framework
+  - release
+category: Release
+---
+
+
+> 💡 The Release candidate series are beta versions with the potential to be a stable version. This means that no major breaking changes are expected until the stable release.
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:30]:
```suggestion
> 💡 Release Candidates are beta versions that have the potential to be released as stable. This means that no major breaking changes are expected until the stable release.
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+Nuxt 3 beta was [announced on October 12, 2021](https://nuxtjs.org/announcements/nuxt3-beta) after 16 months of work, introducing a new foundation based on [Vue 3](https://vuejs.org/), [Vite](https://vitejs.dev/) and [Nitro](https://nitro.unjs.io).
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:32]:
```suggestion
The Nuxt 3 Beta was [announced on October 12, 2021](https://nuxtjs.org/announcements/nuxt3-beta) after 16 months of work, with a new foundation based on [Vue 3](https://vuejs.org/), [Vite](https://vitejs.dev/) and [Nitro](https://nitro.unjs.io).
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+
+Today, 6 months later, we are happy to announce the first release candidate of Nuxt 3, code named “Mount Hope“ 🚀
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:34]:
```suggestion
Today, six months later, we are happy to announce the first release candidate for Nuxt 3, code named “Mount Hope“ 🚀
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+
+Since the beta, we’ve merged [1000+ pull requests](https://github.com/nuxt/framework/pulls?q=is%3Apr+is%3Amerged+created%3A%3E%3D2021-10-12+), closed [800+ issues](https://github.com/nuxt/framework/issues?q=is%3Aissue+is%3Aclosed+sort%3Aupdated-desc+created%3A%3E%3D2021-10-12+), made [2000+ commits](https://github.com/nuxt/framework/commits/main) and seen [3600+ open source repositories](https://github.com/nuxt/framework/network/dependents?package_id=UGFja2FnZS0yMDg1MjMzODUx) using Nuxt.
+
+It’s not just the Nuxt core team; more than [160+ contributors](https://github.com/nuxt/framework/graphs/contributors) have been helping us.
+
+Lastly, we are grateful for seeing amazing adoption by our beta testers: [5000+ stars](https://github.com/nuxt/framework/stargazers) and [340k+ downloads](https://npm-stat.com/charts.html?package=nuxt3) in 6 months 💚
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:40]:
```suggestion
It’s not just the Nuxt core team; more than [160+ contributors](https://github.com/nuxt/framework/graphs/contributors) have been helping us. And we are grateful to see amazing adoption by our beta testers: [5000+ stars](https://github.com/nuxt/framework/stargazers) and [340k+ downloads](https://npm-stat.com/charts.html?package=nuxt3) in 6 months 💚
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+
+You can check the [release notes](https://github.com/nuxt/framework/releases) for migrating from previous versions of Nuxt 3.
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:42]:
```suggestion
You can check the [release notes](https://github.com/nuxt/framework/releases) if you are migrating from a previous version of Nuxt 3.
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+
+Vue 3 has been the [new default for 3 months](https://blog.vuejs.org/posts/vue-3-as-the-new-default.html), bringing better performance, the composition API and TypeScript support. Nuxt 3 builds on these improvements to provide a great Vue 3 experience with first-class SSR support.
+
+Since Nuxt 3 has been rewritten in TypeScript, it is fully typed and provides helpful shortcuts to ensure you have access to accurate type information when you are coding.
+
+On top of this, Nuxt will automatically generate a TS config (`.nuxt/tsconfig.json`) and a global types file (`.nuxt/nuxt.d.ts`) for you to **get a full TypeScript experience with zero configuration**.
+
+Furthermore, you can run the `npx nuxi typecheck` command to manually check your types within your Nuxt application, or even enable build or development type-checking if you prefer.
+
+Read more in the [TypeScript concept](https://v3.nuxtjs.org/guide/concepts/typescript) of Nuxt 3.
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:56]:
```suggestion
Read more about [TypeScript in Nuxt 3](https://v3.nuxtjs.org/guide/concepts/typescript).
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+import { defineNuxtConfig } from 'nuxt'
+
+export default defineNuxtConfig({
+  builder: 'webpack' // default: 'vite'
+})
+```
+
+As part of our deep integration with both Vite and Webpack, we also created [unplugin](https://github.com/unjs/unplugin): a unified plugin system for build tools, which enables writing universal plugins that work in Rollup, Vite and Webpack.
+
+We laid the foundation for making Nuxt a **builder agnostic framework** and preparing it for JavaScript’s perpetually evolving ecosystem.
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:78]:
```suggestion
We've laid the foundation for making Nuxt a **builder agnostic framework** and are preparing it for JavaScript’s perpetually evolving ecosystem.
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 12]
```diff
@@ -0,0 +1,178 @@
...
+
+A note on Vite, we are pushing forward [vite-node](https://www.npmjs.com/package/vite-node) and [Webpack lazyCompilation](https://webpack.js.org/configuration/experiments/#experimentslazycompilation), bringing on-demand bundling for the server bundle of your Nuxt application: No matter how big your application becomes, Nuxt will always start in a few seconds.
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:80]:
```suggestion
A further note on performance: we are pushing forward [vite-node](https://www.npmjs.com/package/vite-node) and [Webpack lazyCompilation](https://webpack.js.org/configuration/experiments/#experimentslazycompilation), bringing on-demand bundling for the server bundle of your Nuxt application: No matter how big your application becomes, Nuxt will always start in a few seconds.
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 13]
```diff
@@ -0,0 +1,178 @@
...
+
+## Nitro & UnJS
+
+One of the major features of Nuxt 3 is its new server engine [UnJS/Nitro](https://nitro.unjs.io/) making Nuxt a powerful full stack and provider-agnostic framework.
+
+### Why Nitro?
+
+First pain point in Nuxt 2 was the lack of server routes, we had [serverMiddleware](https://nuxtjs.org/docs/configuration-glossary/configuration-servermiddleware) but the developer experience was not great. It lacked alias support and was unstable.
+
+Secondly, the production server was not optimized to work in a lambda environment, boot time was slow and install size was big.
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:90]:
```suggestion
Secondly, the production server was not optimized to work in a lambda environment; boot time was slow and install size was big.
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 14]
```diff
@@ -0,0 +1,178 @@
...
+
+A note on Vite, we are pushing forward [vite-node](https://www.npmjs.com/package/vite-node) and [Webpack lazyCompilation](https://webpack.js.org/configuration/experiments/#experimentslazycompilation), bringing on-demand bundling for the server bundle of your Nuxt application: No matter how big your application becomes, Nuxt will always start in a few seconds.
+
+## Nitro & UnJS
+
+One of the major features of Nuxt 3 is its new server engine [UnJS/Nitro](https://nitro.unjs.io/) making Nuxt a powerful full stack and provider-agnostic framework.
+
+### Why Nitro?
+
+First pain point in Nuxt 2 was the lack of server routes, we had [serverMiddleware](https://nuxtjs.org/docs/configuration-glossary/configuration-servermiddleware) but the developer experience was not great. It lacked alias support and was unstable.
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:88]:
```suggestion
The first pain point in Nuxt 2 was a lack of robust server integration. We had [serverMiddleware](https://nuxtjs.org/docs/configuration-glossary/configuration-servermiddleware) but the developer experience was not great. It lacked alias support and was unstable.
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 15]
```diff
@@ -0,0 +1,178 @@
...
+Nitro takes Nuxt to another level with multiple features:
+
+- Hot module replacement for the [server/ directory](https://v3.nuxtjs.org/guide/features/server-routes)
+- Server routes with file system similar as the pages/ directory
+- Portable and compact deployments without node\_modules dependency
+- Route caching and static pre-rendering with built-in crawler
+- Code-splitting and async chunk loading for fast server startup time
+- [And much more…](https://nitro.unjs.io/)
+
+**This means that Nuxt 3 and everything you use in your project are now [devDependencies](https://nodejs.dev/learn/npm-dependencies-and-devdependencies)**, Nitro will take care of code-splitting and bundling everything into a portable and compact `.output` directory ready to be deployed on [any hosting providers](https://nitro.unjs.io/deploy/).
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:101]:
```suggestion
**This means that Nuxt 3 and everything you use in your project are now [devDependencies](https://nodejs.dev/learn/npm-dependencies-and-devdependencies)**. Nitro will take care of code-splitting and bundling everything into a portable and compact `.output` directory ready to be deployed to [any hosting provider](https://nitro.unjs.io/deploy/).
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 16]
```diff
@@ -0,0 +1,178 @@
...
+
+Here is a [small video](https://twitter.com/Atinux/status/1506597209537658885) about how it looks.
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:103]:
```suggestion
Here is a [small video](https://twitter.com/Atinux/status/1506597209537658885) showing what it looks like.
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 17]
```diff
@@ -0,0 +1,178 @@
...
+  type: video/mp4
+- src: https://res.cloudinary.com/nuxt/video/upload/v1650468764/nuxt3-output_iflx7y.ogv
+  type: video/ogg
+poster: https://res.cloudinary.com/nuxt/video/upload/v1650468764/nuxt3-output_iflx7y.jpg
+---
+::
+
+### Unified JavaScript Tools
+
+In 2018, we started creating reusable and framework agnostic JavaScript libraries such as [unjs/consola](https://github.com/unjs/consola) and [unjs/ipx](https://github.com/unjs/ipx). These libraries are small, well-tested, and compatible with each other to benefit the entire JS ecosystem. Today we have a bigger collection of them in a new GitHub organization called[ ](https://github.com/unjs)[UnJS](https://github.com/unjs). This organization is made for offering a set of [ESM friendly](https://v3.nuxtjs.org/guide/going-further/esm) JavaScript tools suitable for running in any JavaScript environment including Node.js, Deno, Browsers and Workers.
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:120]:
```suggestion
In 2018, we started creating reusable and framework-agnostic JavaScript libraries such as [unjs/consola](https://github.com/unjs/consola) and [unjs/ipx](https://github.com/unjs/ipx). These libraries are small, well-tested, and compatible with each other to benefit the entire JS ecosystem. Today we have a bigger collection of them in a new GitHub organization called [UnJS](https://github.com/unjs). This organization is made for offering a set of [ESM friendly](https://v3.nuxtjs.org/guide/going-further/esm) JavaScript tools suitable for running in any JavaScript environment including Node.js, Deno, Browsers and Workers.
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 18]
```diff
@@ -0,0 +1,178 @@
...
+
+By working on this foundation for over 4 years, we are progressively making Nuxt capable of running everywhere and exploring ideas beyond our imagination! 
+
+Checkout <https://github.com/unjs> to discover all the packages powering Nuxt 3.
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:124]:
```suggestion
Check out <https://github.com/unjs> to discover all the packages powering Nuxt 3.
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 19]
```diff
@@ -0,0 +1,178 @@
...
+
+# Powerful features
+
+## File System Automation
+
+### Pages
+
+It all started with the pages directory: every file is mapped to a route. We were the first in the frontend framework ecosystem to support dynamic and nested routes thanks to the amazing work done by the Vue team on [vue-router](https://router.vuejs.org/).
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:132]:
```suggestion
It all started with the pages directory: every file is mapped to a route. We were the first in the frontend framework ecosystem to support dynamic and nested routes, thanks to the amazing work done by the Vue team on [vue-router](https://router.vuejs.org/).
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 20]
```diff
@@ -0,0 +1,178 @@
...
+
+The [pages directory](https://v3.nuxtjs.org/guide/directory-structure/pages) remains with an improved syntax for [dynamic routes](https://v3.nuxtjs.org/guide/directory-structure/pages#dynamic-routes), allowing to have multiple parameters in a single route. Moreover, the pages directory is optional! You can start your project with only an [app.vue](https://v3.nuxtjs.org/guide/directory-structure/app) file and Nuxt will use a minimal universal router instead for an optimized bundle size (saving 28kB, 21% tinier out of the box).
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:134]:
```suggestion
The [pages directory](https://v3.nuxtjs.org/guide/directory-structure/pages) remains, featuring an improved syntax for [dynamic routes](https://v3.nuxtjs.org/guide/directory-structure/pages#dynamic-routes) that allows multiple parameters in a single route. Moreover, the pages directory is optional! You can start your project with only an [app.vue](https://v3.nuxtjs.org/guide/directory-structure/app) file and Nuxt will use a minimal universal router instead for an optimized bundle size (saving 28kB, 21% tinier out of the box).
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 21]
```diff
@@ -0,0 +1,178 @@
...
+
+Take a look at the [migration guide for pages](https://v3.nuxtjs.org/migration/pages-and-layouts#pages) for more details.
+
+### Plugins
+
+Every file in the [plugins directory](https://v3.nuxtjs.org/guide/directory-structure/plugins) will be automatically imported and run before creating the main component ([app.vue](https://v3.nuxtjs.org/guide/directory-structure/app)).
+
+### Components
+
+The [components directory](https://v3.nuxtjs.org/guide/directory-structure/components) is another piece of art, every component inside will be available in your templates, no imports required. Nuxt will then analyze your code to only include the components used in the final bundle, bringing both great developer experience and performance for production.
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:144]:
```suggestion
The [components directory](https://v3.nuxtjs.org/guide/directory-structure/components) is another piece of art; every component inside will be available in your templates, no imports required. Nuxt will then analyze your code, only including the components you use in the final bundle, bringing both great developer experience and performance for production.
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 22]
```diff
@@ -0,0 +1,178 @@
...
+
+### Composables
+
+Thanks to Vue 3 and the Composition API, we created a new [composables directory](https://v3.nuxtjs.org/guide/directory-structure/composables) to automatically import your Vue composables into your application.
```
Comment (danielroe) [inline content/en/announcements/6.nuxt3-rc.md:148]:
```suggestion
Thanks to Vue 3 and the Composition API, we've created a new [composables directory](https://v3.nuxtjs.org/guide/directory-structure/composables) to automatically import your Vue composables into your application.
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 23]
```diff
@@ -0,0 +1,176 @@
...
+
+```ts
+import { defineNuxtConfig } from 'nuxt'
+
+export default defineNuxtConfig({
+  builder: 'webpack' // default: 'vite'
+})
+```
+
+As part of our deep integration with both Vite and Webpack, we also created [unplugin](https://github.com/unjs/unplugin): a unified plugin system for build tools, which enables writing universal plugins that work in Rollup, Vite and Webpack.
```
Comment (pi0) [inline content/en/announcements/6.nuxt3-rc.md:74]:
```suggestion
As part of our deep integration with both Vite and Webpack, we also created [unjs/unplugin](https://github.com/unjs/unplugin): a unified plugin system for build tools, which enables writing universal plugins that work in Rollup, Vite and Webpack.
```

File: content/en/announcements/6.nuxt3-rc.md [hunk 24]
```diff
@@ -0,0 +1,176 @@
...
+
+We've laid the foundation for making Nuxt a **builder agnostic framework** and are preparing it for JavaScript’s perpetually evolving ecosystem.
+
+A further note on performance: we are pushing forward [vite-node](https://www.npmjs.com/package/vite-node) and [Webpack lazyCompilation](https://webpack.js.org/configuration/experiments/#experimentslazycompilation), bringing on-demand bundling for the server bundle of your Nuxt application: No matter how big your application becomes, Nuxt will always start in a few seconds.
+
+## Nitro & UnJS
+
+One of the major features of Nuxt 3 is its new server engine [UnJS/Nitro](https://nitro.unjs.io/) making Nuxt a powerful full stack and provider-agnostic framework.
```
Comment (pi0) [inline content/en/announcements/6.nuxt3-rc.md:82]:
```suggestion
One of the major features of Nuxt 3 is its new server engine [unjs/nitro](https://nitro.unjs.io/) making Nuxt a powerful full stack and provider-agnostic framework.
```

**Pull request merged**